### ISSUE TITLE
Component registration incorrectly includes non-SVG files

### ISSUE THREAD

**Issue by simensen** [driesvints/blade-icons#95]
- Library Version: 0.4.5
- Laravel Version: 8.1.0
- PHP Version: 7.4.6

### Description:

When I specify a directory for a set that contains mixed SVG and JPEG or PNG, components/aliases are registered for all files and not just the SVG files.

### Steps To Reproduce:

Add a JPEG or PNG to a directory covered by a set. Try using `x-[name-of-jpeg-or-png]`. A component is registered with the expected name, but the error output is displayed because at the point of rendering it attempts to look for the `.svg` extension and it is not found.

For example, if a file named `event-storming-crumpled.jpg` exists inside `resources/images`, along with `logo.svg`, and I point the path to `resources/images` with prefix `svg`, both `x-svg-logo` and `x-svg-event-storming-crumpled` are registered. However, when trying to render `<x-svg-event-storming-crumpled />`, the following is thrown:

```
Svg by name "event-storming-crumpled" from set "default" not found. (View: /var/www/resources/views/layouts/admin.blade.php)
```

Since the component is smart enough to only open `.svg` extension, it seems like component registration process should be able to filter for `.svg` and only register component aliases for SVG files.

This seems like a bug to me, but I could see how it could be considered a feature request. If this bug is confirmed as such, I'd consider trying to fix this myself.

**Comment by driesvints** [comment]
Yeah, I think it makes sense to filter for only SVG icons. Feel free to PR otherwise I'll take a look when I find time. Thanks for the report 👍

**Comment by driesvints** [comment]
Will be in the next release.

**Comment by driesvints** [comment]
This one was fixed in 0.5.1
